Governance & Compliance (Are they good?)/Privacy Policy & Terms of Service/Terms of Service and Privacy Policy documents.yaml

```diff
@@ -13,6 +13,11 @@ criterias:
           The ToS are available in the language(s) most commonly spoken by the
           company's users.
 
+          Privacy policies and ToS are accessible through the company's websites and are available for search engines to process.
+
+          Privacy policies and ToS are accessible via a stable URL.
+
+          Privacy policies and ToS for specific devices are available online.
```

Comment:
"products and services"

Try to say, "it's clear which piece of documentation applies to the product/service."